### ISSUE TITLE
Retinoschisis

### ISSUE THREAD

**Issue by obophenotype-user** [obophenotype/mammalian-phenotype-ontology#713]
-------- Original Message --------
Subject:    MP term
Date:   Tue, 8 Jun 2010 16:32:51 -0400
From:   Melissa Berry &lt;Melissa.Berry@jax.org&gt;
To:     Cynthia Smith &lt;csmith@informatics.jax.org&gt;

Hi Cindy,

Retinoschisis (“an eye disease characterized by the abnormal splitting 
of the retina's neurosensory layers, usually in the outer plexiform 
layer, resulting in a loss of vision in the corresponding visual field”)
Child of 6069

Note to self: J:160257

Reported by: cindyJax

Original Ticket: [obo/mammalian-phenotype-requests/713](https://sourceforge.net/p/obo/mammalian-phenotype-requests/713)
